src/SlashCommands.tsx

```diff
@@ -286,6 +287,43 @@ export const Commands = [
         category: CommandCategories.admin,
         renderingTypes: [TimelineRenderingType.Room],
     }),
+    new Command({
+        command: 'jumptodate',
+        args: '<date>',
+        description: _td('Jump to the given date in the timeline (YYYY-MM-DD)'),
+        isEnabled: () => SettingsStore.getValue("feature_jump_to_date"),
+        runFn: function(roomId, args) {
+            if (args) {
+                return success((async () => {
+                    const unixTimestamp = Date.parse(args);
+                    if (!unixTimestamp) {
+                        throw new Error(
+                            `We were unable to understand the given date (${args}). Try using the format YYYY-MM-DD.`,
```

Comment:
I think we should add the i18n `_t(...)` stuff to this so people can translate 🤔

Reply:
Wonder if we can make use of `newTranslatableError(_td(...));` - we'd probably need to update SlashCommands to expect a translatable error

Reply:
> Without using this the error in rageshake would be translated too

I've used `_t` for now 😬

---

It looks like `newTranslatableError` also needs to be updated to allow arguments.

Can we handle this in a separate PR?

Reply:
Sure, would be good to drop a comment inline to remind us of that (somewhat)

Reply:
> Sure, would be good to drop a comment inline to remind us of that (somewhat)

Will work on that PR now ⏩

All good to merge still?

Reply:
Yup!